unimplemented/00070-apply-sp-using-ssm.md

```diff
@@ -0,0 +1,39 @@
+- Feature Name: Apply Service Packs to SLES and/or openSUSE Leap using SSM
+- Start Date: 2023-03-10
+
+# Summary
+[summary]: #summary
+
+Provice an easy mechanism to upgrade SLES and/or openSUSE Leap systems from one service pack the next one
```

Comment:
I think we already this mechanism in place: https://documentation.suse.com/suma/4.2/en/suse-manager/client-configuration/client-upgrades-product-migration.html

What we would need is to have support to make in more then one minion at a time (batch SP migration)